patches/rpm-4.15.1-fix-rpmbuild-segfault.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/build/pack.c b/build/pack.c
```

Comment:
Maybe add a comment about the need and purpose of the patch? Either in the diff file (I think the first lines get ignored for example) or in the PR description